coderd/audit.go

```diff
@@ -198,7 +198,7 @@ func (api *API) convertAuditLog(ctx context.Context, dblog database.GetAuditLogs
 			Username:  dblog.UserUsername.String,
 			Email:     dblog.UserEmail.String,
 			CreatedAt: dblog.UserCreatedAt.Time,
-			Status:    codersdk.UserStatus(dblog.UserStatus),
+			Status:    codersdk.UserStatus(dblog.UserStatus.UserStatus),
```

Comment:
Userstatus can be null?? This seems incorrect? 

I assume it is because we do a `join` and the user might not exist? 
https://github.com/coder/coder/blob/7ea8efdd794469fa7afd14551194904400ee68a8/coderd/database/queries/auditlogs.sql#L16-L16

But if that is the case, why is `status` the only nullable field?

Reply:
Ok so it is a change to SQLc since joins can yield null columns. 

We can keep this as I am seeing all the columns from the join be nullable. These are potentially `null` because the user row in the `users` table might not exist any more.

---

Just a note:

In a case we are doing joins with non-nullable columns (eg because of foreign key constraints), we can use `COALESCE` to remove the nullablity of the column. Or we can use `INNER` joins. 

Eg: https://play.sqlc.dev/p/64da08f9cd4b8fb8dc49fe20da5c9b6e0de804ee6399bf5b3abb923f317bbe00